addons/explosives/functions/fnc_defuseExplosive.sqf

```diff
@@ -17,7 +17,9 @@
 #include "script_component.hpp"
 EXPLODE_2_PVT(_this,_unit,_explosive);
 
-if (getNumber (ConfigFile >> "CfgAmmo" >> typeof _explosive >> "ACE_explodeOnDefuse") == 1) exitWith {
+_explode = random 1.0;
+if (getNumber (ConfigFile >> "CfgAmmo" >> typeOf _explosive >> "ACE_explodeOnDefuse") == 1 &&
+	_explode < GVAR(ExplodeOnDefuseChance)) exitWith {
```

Comment:
Certain explosives like the SLAM have safety features built in, which cause them in certain circumstances to detonate on tampering attempts. I'd very much like to maintain that functionality as it replicates reality.

However, for IEDs there is room for variability. So if you can allow both behaviours. All the better.